core/bindings.rs

```diff
@@ -0,0 +1,549 @@
+// Copyright 2018-2020 the Deno authors. All rights reserved. MIT license.
+
+#![allow(mutable_transmutes)]
+#![allow(clippy::transmute_ptr_to_ptr)]
+
+use crate::isolate::Isolate;
+use crate::libdeno::deno_buf;
+use crate::libdeno::script_origin;
+use crate::libdeno::PinnedBuf;
+
+use rusty_v8 as v8;
+use v8::InIsolate;
+
+use libc::c_char;
+use libc::c_void;
+use std::convert::TryFrom;
+use std::ffi::CString;
+use std::option::Option;
+
```

Comment:
consider moving EXTERNAL_RESOURCES definition to this module, since I think it is exactly a list of these "bindings"